### PR TITLE
Update manager capabilities to 3.22 to fix deprecation in Ember 3.26

### PR DESCRIPTION
**Pull request by cibernox**
This is flooding my logs with thousands of warnings when using ember beta (currently 3.26).

### CODE REVIEW

File: addon/modifiers/did-insert.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
+    capabilities: capabilities('3.22', { disableAutoTracking: true }),
```
Comment (rwjblue) [inline addon/modifiers/did-insert.js:49]: Needs to use `gte` to use the capabilities as available by Ember version

See https://github.com/emberjs/ember-render-modifiers/pull/25#pullrequestreview-510614931

**Comment by simonihmig** [comment]
> This is flooding my logs with thousands of warnings

Same here :) @cibernox would you be able to address the PR review here?

**Comment by cibernox** [comment]
I missed the feedback completely on this one. I just fixed it.

**Comment by cibernox** [comment]
Not sure why did-update is failing for ember 3.24+

**Comment by sandydoo** [comment]
@cibernox, probably related to autotracking. `did-update` doesn’t “consume” the arguments on initial render, so the update hook isn’t run. The hack-du-jour is to loop over all of the arguments, like in this previous attempt to solve this same issue https://github.com/emberjs/ember-render-modifiers/pull/25#discussion_r507393325.
It looks like the Ember team decided to actually use the workaround in the meantime (https://github.com/ember-modifier/ember-modifier/pull/63#issuecomment-815908201), but things have stalled…

**Comment by boris-petrov** [comment]
That's also what we did in [ember-render-helpers](https://github.com/buschtoens/ember-render-helpers/pull/286).

File: addon/modifiers/did-update.js [hunk 2]
```diff
@@ -64,9 +64,18 @@ export default setModifierManager(
     createModifier() {
       return { element: null };
     },
-    installModifier(state, element) {
+    installModifier(state, element, args) {
       // save element into state bucket
       state.element = element;
+      if (gte('3.22.0')) {
+        // consume all args
```
Comment (sandydoo) [inline addon/modifiers/did-update.js:71]: @cibernox, you have to do this in `updateModifier` as well

Comment (cibernox) [reply addon/modifiers/did-update.js:71]: Done!

**Comment by jgadbois** [comment]
@simonihmig Is there anything remaining to get this merged?

**Comment by simonihmig** [comment]
> @simonihmig Is there anything remaining to get this merged?

Well, I am neither the author nor one of the maintainers, so it's not really up to me! 🙂 
But AFAICT it's just that one test scenario that is still failing...

**Comment by jgadbois** [comment]
> > @simonihmig Is there anything remaining to get this merged?
> 
> Well, I am neither the author nor one of the maintainers, so it's not really up to me! 
> But AFAICT it's just that one test scenario that is still failing...

Whoops!  I'll see if i can look at that test scenario later.  The deprecation warnings are getting pretty intense :)

**Comment by cibernox** [comment]
@rwjblue I had to pass `disableAutoTracking: false` (check https://github.com/emberjs/ember-render-modifiers/pull/33/commits/7146c3ab4440aa853a257fcd20779921403d04d8 ) to make tests pass in 3.24.

Tests continue to hang forever in release / beta / canary without anything being logged. I recall IE11 being to blame on this, but I removed it from the targets and it's still hanging. Any idea what might be causing it?

**Comment by jgadbois** [comment]
Hi @cibernox thanks for working on this - have you made any progress on the remaining tests failing?

**Comment by btecu** [comment]
@cibernox why the switch from `yarn` to `npm`?
Was it necessary to drop IE support? That will likely require a major release.

**Comment by rwwagner90** [comment]
@cibernox @rwjblue bump, this is really necessary for all modern Ember versions.

**Comment by cibernox** [comment]
@rwwagner90 I'm a bit short of time right now to push thing forward, maybe someone can take over where I left? Tests were hanging forever and I don't recall the reason (or even if I knew the reason at some point)

**Comment by SergeAstapov** [comment]
I re-created this as #42 which needs to land after #40 in order to get CI happy (see #42 description for more info about CI failures).

**Comment by jgadbois** [comment]
Awesome, thank you so much!

**Pull request closed without merging**